articles/css-in-js-solutions/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/css-in-js-solutions/
+title: CSS-In-JS For Web Components
+description: This article will cover various techniques a developer can use in defining styles within app components during development than maintaining multiple stylesheets for a single app component.
+author: aransiola-ayodele-leom
+date: 2020-11-11T00:00:00-16:00
+topic: []
+excerpt_separator: <!--more-->
+images:
+
+    - url: /engineering-education/css-in-js-solutions/hero.jpg
+      alt: code view on a laptop
+---
+We now write the modern web in components and not pages. They designed CSS to declare styles in different files (.CSS) for each page of your web application, and this structure makes it challenging to maintain multiple stylesheets for multiple pages.
+<!--more-->
+
+### CSS-In-JS For Web Components
+Web components have changed the landscape for the development of web applications for front-end developers. Using web components provides ample opportunities to reuse and build UI components and custom elements critical in the development process of web applications.
```

Comment:
changed the landscape of **web development** **would flow better**

development process**.** **would flow better**